### DISCUSSION TITLE
How to create table from the schema from parquet file

### DISCUSSION THREAD

**Discussion by kesavkolla** [databendlabs/databend#8985] [Q&A] [answered]
We have pretty complex schema in parquet file. Is it possible to dervice the schema from parquet and create table automatically in databend?

**Comment by bohutang** [comment, marked as answer]
Yes, the infer schema RFC: https://github.com/datafuselabs/databend/pull/8645

And Datahend already have the infer_schema:
https://github.com/datafuselabs/databend/blob/f4a4ea8583a950790daef1181a384d6cb8b1f4d8/src/query/pipeline/sources/src/processors/sources/input_formats/impls/input_format_parquet.rs#L91

The first is to make Databend read patquet file and querying them directly as a query engine, then infer the schema to fuse table to fast the query.
